lib/view_component/with_content_helper.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module ViewComponent
+  module WithContentHelper
+    def with_content(value = nil, &block)
```

Comment:
Since we only support strings, how do you feel about removing block support to focus `with_content` even more for v1?

Reply:
I agree ☝🏻

Reply:
done in https://github.com/github/view_component/pull/875/commits/8833c4b776820115d112d615377a2f80e36db9cd